src/activities/datastore/activities.js

```diff
@@ -136,6 +138,9 @@ export default {
         dispatch('fetchListByGroupId', groupId)
       }
     },
+    async dismissFeedback ({ commit, rootGetters }, activityId) {
+      await activities.dismissFeedback(activityId)
```

Comment:
@larzon83 Could this call also be made asynchronously, i.e. without the await?

Reply:
Yes, that is possible (in this case): What's your motivation behind this?

Reply:
We are not processing the request's response, but having await might give the impression that it should be processed. Other developers could be confused by this. So reducing sources for potential future confusion is by motivation.

Reply:
Yes, thats correct. It's ok for me to remove the async. Maybe we can leave this comment open for Nick to check in too on this.

Reply:
I think it's nice with the `await`, it means the vuex action will only be completed after the request is done, I think that is a nice principle.

If removing the `async/await` then returning the result of the API call (a promise) would also achieve that, but the `await` is more consistent IMO.

Reply:
... also I guess you decided to rely on the websocket update to keep the data updated.

Personally, I still update from the request response, but I'm also very happy with it like this - happy to revisit this in a more general way when rethinking the data flow with vue3 and the composition API stuff...

Reply:
When this action gets dispatched from the button, we don't process the returned result. So, I think we could remove the async/await and let the action do its thing – I tested it and it works, too. When the backend has finished its stuff, the frontend gets updated via websocket...
Could we even get a returned value with this `actionsToEvents` thing?
Anyway, I myself would also vote for keeping async/await. Also for consistency with other actions.

And yes, we choose the websocket way to keep the data updated. We were lazy refactoring the backend to return the processed activity instead of returning only the status code 😄